dags/utils/gcp.py

```diff
@@ -439,11 +442,12 @@ def export_to_parquet(
                 bucket_name=gcs_output_bucket,
                 prefix=avro_prefix,
                 gcp_conn_id=gcp_conn_id,
+                trigger_rule=trigger_rule.TriggerRule.ALL_DONE,
             )
             avro_export >> run_dataproc_pyspark >> avro_delete
 
-        create_dataproc_cluster >> run_dataproc_pyspark >> delete_dataproc_cluster
-        run_dataproc_pyspark >> gcs_to_s3
+        create_dataproc_cluster >> run_dataproc_pyspark >> gcs_to_s3
+        gcs_to_s3 >> delete_dataproc_cluster
```

Comment:
This can be merged into a single line:

```suggestion
        create_dataproc_cluster >> run_dataproc_pyspark >> gcs_to_s3 >> delete_dataproc_cluster
```

Reply:
it can, but it's longer than 88 characters